dialog/src/main/java/roboy/dialog/states/gameStates/GamingTwentyQuestionsState.java

```diff
@@ -253,7 +280,7 @@ private Output processUserGuessAnswer(String intent){
 
 	private void resetGame(){
 
-		aw = new AkiwrapperBuilder().setFilterProfanity(true).build();
+		startAW(true);
```

Comment:
Is this really what you want to do, is it even required that we reset AW. Can you get to this state without AW being set